<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>10</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>12</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Script MT Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>My Shopping Cart</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="itemWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Product ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Color</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Discount(if any)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="total">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkoutButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Order Now !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue Shopping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
